### DISCUSSION TITLE
Use Cases / Praise

### DISCUSSION THREAD

**Discussion by MetallicDoritos** [jely2002/youtube-dl-gui#524] [General]
### Leave a comment if you find other sites that work with OVD!
I am unsure if this has been previously mentioned, but I've successfully used this program for _more than just YT videos_. It works with sites such as: 1. [Bandcamp](https://bandcamp.com/), 2. [Instagram](https://www.instagram.com/), 3. [DailyMotion](https://www.dailymotion.com/), 4. [PeerTube](https://peertube.tv/), 5. [Soundcloud](https://soundcloud.com/), 6. [TikTok](https://www.tiktok.com/), 7. [Vimeo](https://vimeo.com/)
_...and possibly many others_ outside of this list! I was honestly surprised at how useful this can be for _archiving_ videos as well as _sampling for audio production_ and _listening to music offline_ (no ads, yay!).
This version _isn't updated_ and **won't** update on it's own, but this fork by [Stefan Lobbenmeier](https://github.com/StefanLobbenmeier/youtube-dl-gui) has **_worked flawlessly_** for months, so **thank you** for your contribution!
This is _truly awesome software_ that I will continue sharing and vetting as long as it works.

**Comment by EliRoberts5** [comment]
I am just using this for youtube so far and it is very impressive. yt-dlp.exe and youtube-dl-gui ( and youtube-dl.exe maybe ), this is a great app so far.  Can you imagine when we get the linux subsystem for android on android 16 we could potentially run it on an android phone ..
